boards/telosb/board.c

```diff
@@ -67,7 +67,7 @@ void msp430_init_dco(void)
     BCSCTL1 |= DIVA1 + DIVA0;             /* ACLK = LFXT1CLK/8 */
 
     for (i = 0xFFFF; i > 0; i--) {        /* Delay for XTAL to settle */
-        asm("nop");
+        __asm__("nop");
```

Comment:
Yes, like there
On Mar 31, 2016 10:33 PM, "Kaspar Schleiser" <notifications@github.com>
wrote:

> In boards/telosb/board.c
> <https://github.com/RIOT-OS/RIOT/pull/5059#discussion_r58122804>:
>
> > @@ -67,7 +67,7 @@ void msp430_init_dco(void)
> >      BCSCTL1 |= DIVA1 + DIVA0;             /* ACLK = LFXT1CLK/8 */
> >
> >      for (i = 0xFFFF; i > 0; i--) {        /* Delay for XTAL to settle */
> > -        asm("nop");
> > +        __asm__("nop");
>
> @gebart <https://github.com/gebart> you mean like here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/5059/files/c3f7186d4e686561b3e5714d91819d1b361542cd#r58122804>
>